src/transformers/models/dpt/configuration_dpt.py

```diff
@@ -221,14 +222,17 @@ def __init__(
             self.backbone_config = backbone_config
             self.backbone_featmap_shape = None
             self.neck_ignore_stages = []
-
         else:
             self.backbone_config = backbone_config
             self.backbone_featmap_shape = None
             self.neck_ignore_stages = []
 
+        if use_autobackbone and backbone_config is not None and backbone is not None:
```

Comment:
Moved to account for the recent DPT updates where backbone and backbone_config can be passed in, but aren't necessary.